### PR TITLE
Add support for downloading bridge lists from ooni-resources

### PR DESCRIPTION
**Pull request by hellais**
This implements what is described in: #652

### CODE REVIEW

File: ooni/deck/store.py [hunk 1]
```diff
@@ -51,30 +73,54 @@ def update_url_lists(self, country_code):
...
+
+        data_fname = "tor-bridge-lines.txt"
+        desc_fname = "tor-bridge-lines.desc"
+        out_file = self.path.child("data").child(data_fname)
+
+        def format_row(row):
+            host, port, nickname, protocol = row
+            if protocol in ("ssh", "https", "port-444"):
+                return "{}:{}\n".format(host, port)
+            return "{} {}:{}\n".format(protocol, host, port)
```
Comment (darkk) [inline ooni/deck/store.py]: I understand, that it's required to _parse_ `protocol` from the datafile in `setUp` (compatibility with old lists in the moment of upgrade as the list will be stale for an hour or so), but I don't understand the reason to drop `protocol` selectively in `format_row`.

File: ooni/deck/store.py [hunk 2]
```diff
@@ -21,6 +21,28 @@ class InputNotFound(Exception):
 class DeckNotFound(Exception):
     pass
 
+
+def write_txt_from_csv(in_file, out_file, func, skip_header=True):
+    out_fh = out_file.open('w')
```
Comment (hellais) [inline ooni/deck/store.py]: At this point there is actually no real valid reason for not doing what you suggest. I guess I got into the bad habbit of doing the above pattern to maintain backward compatibility with python 2.6 that didn't support multiple with statements on the same line.

I will adjust accordingly.

File: ooni/deck/store.py (hunk 1 above)
Comment (hellais) [inline ooni/deck/store.py]: The reason for dropping the protocol at the beginning is that we care to indicate the protocol only in the case that it's a pluggable transport type.

In other cases we should just treat it as a non pluggable transport bridge and go ahead with a regular bridge test.

Comment (darkk) [inline ooni/deck/store.py]: I see. I'd rather reuse list of known `pluggable_transports` from `tcp_connect` to make it explicit and cross-referenced...
Also, ordinary bridges deserve tor-handshake but not tcp-handshake (I assume, you meant that too), but I see no public old-school bridge [in the TBB source](https://gitweb.torproject.org/builders/tor-browser-bundle.git/tree/Bundle-Data/PTConfigs/bridge_prefs.js).
Anyway, that's not a big deal, I got your point.

**Pull request merged**